polimec-common/common/src/credentials/mod.rs

```diff
@@ -49,41 +51,48 @@ parameter_types! {
 	pub const Institutional: InvestorType = InvestorType::Institutional;
 }
 
-#[derive(Clone, Encode, Decode, Eq, PartialEq, Ord, PartialOrd, RuntimeDebug, TypeInfo, Deserialize, Serialize)]
+#[derive(Clone, Encode, Decode, Eq, PartialEq, Ord, PartialOrd, RuntimeDebug, TypeInfo, Deserialize)]
 pub struct SampleClaims<AccountId> {
 	#[serde(rename = "sub")]
 	pub subject: AccountId,
 	#[serde(rename = "iss")]
-	pub issuer: scale_info::prelude::string::String,
+	pub issuer: String,
 	pub investor_type: InvestorType,
+	#[serde(deserialize_with = "from_bounded_vec")]
+	pub did: DID,
 }
 
-pub struct EnsureInvestor<T, I, Type>(sp_std::marker::PhantomData<(T, I, Type)>);
-impl<'de, T, I, Type> EnsureOriginWithCredentials<T::RuntimeOrigin> for EnsureInvestor<T, I, Type>
+pub type DID = BoundedVec<u8, ConstU32<57>>;
```

Comment:
One small this. I think the naming convention for types should be PascalCase so might be good to change it to Did :)

Reply:
But at the same time its an abbreviation so might be actually correct casing haha. Okay skip my previous remark 🤡

Reply:
![CleanShot 2024-03-15 at 11 44 47@2x](https://github.com/Polimec/polimec-node/assets/54085674/6764450f-bff8-4a85-b0c8-8431629c7752)

ChatGPT agreed with you tho, which is why I changed it lol